File: java/src/test/java/codingdojo/CalcStats3Test.java
package codingdojo;

import org.junit.jupiter.api.Test;

import java.util.List;
import java.util.stream.Stream;

import static org.junit.jupiter.api.Assertions.assertNotNull;

public class CalcStats3Test {

    @Test
    void getReport() {
        var values = List.of(56, 2, 41);
        var report = new CalcStats3(values).getReport();
        assertNotNull(report);
    }
}

File: php/test/CalcStats3Test.php
<?php

namespace Tests;

use CodingDojo\CalcStats3;
use PHPUnit\Framework\TestCase;

class CalcStats3Test extends TestCase
{
    public function testGetReport()
    {
        $values = [56, 2, 41];
        $calcStats3 = new CalcStats3($values);
        $report = $calcStats3->getReport();
        $this->assertNotNull($report);
    }
}
